webapp/i18n/fr.json

```diff
@@ -38,7 +38,7 @@
   "ContentBlock.moveUp": "Déplacer vers le haut",
   "ContentBlock.text": "texte",
   "DashboardPage.CenterPanel.ChangeChannels": "Utiliser le sélecteur pour changer de canal",
-  "DashboardPage.title": "Bienvenue dans Focalboard (Beta) !",
```

Comment:
We can merge it, but the source of truth for the translations is the translation platform, not the github repo, so probably is going to be reverted by the PR for new translations. Just to keep in mind this needs to be change there too.

Reply:
Thanks, I've made the corresponding changes in Weblate. Please check if I submitted them correctly: https://translate.mattermost.com/projects/focalboard/webapp/#history, thanks.